linux/network_device_ops/apply_new_network_config.yml

```diff
@@ -156,7 +156,10 @@
 # Get IP address of new interface
 - name: "Set fact of searching IPv4 address"
   ansible.builtin.set_fact:
-    search_ip: "{% if new_nic_ipv4 is defined and new_nic_ipv4 %}{{ new_nic_ipv4 }}{% else %}192.168{% endif %}"
+    search_ip: >-
+      {%- if new_nic_ipv4_method == 'static' -%}{{ new_nic_ipv4 }}
+      {%- else -%}192.168
```

Comment:
I think here can replace `192.168` with `{{ network_env_gateway | regex_search('^\d+(\.\d+){2}') }}`

Reply:
I think here can leave it since it's hard coded.